<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>148</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralwidget{
background-image: url(c:/tvip/assets/Fondo-Login.png);
 background-repeat: no-repeat;
}</string>
  </property>
  <widget class="QPushButton" name="botonGuardar">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>30</y>
     <width>85</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(10, 137, 103);
border-radius:10px;
font: 75 11pt &quot;Bahnschrift Condensed&quot;;</string>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botonGuardar_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>80</y>
     <width>85</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(10, 137, 103);
border-radius:10px;
font: 75 11pt &quot;Bahnschrift Condensed&quot;;</string>
   </property>
   <property name="text">
    <string>CANCELAR</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <item>
    <property name="text">
     <string>Enviar MSJ</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Actualizaciones</string>
    </property>
   </item>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
